components/brave_wallet_ui/page/screens/send/send_screen/send_screen.tsx

```diff
@@ -86,6 +90,9 @@ import {
 import {
   SelectAddressButton //
 } from '../../composer_ui/select_address_button/select_address_button'
+import { AddMemo } from '../components/add_memo/add_memo'
+
+export const MAX_MEMO_LENGTH = 500
```

Comment:
nit - move this to a better place and call it `MAX_ZCASH_MEMO_LENGTH`

Reply:
Moved to a `constants` folder in the `send` directory.